checker/bin-devel/build.sh

```diff
@@ -31,14 +31,14 @@ else
 fi
 
 # Clone the annotated JDK into ../jdk .
-"$PLUME_SCRIPTS/git-clone-related" typetools jdk
+"$PLUME_SCRIPTS/git-clone-related" typetools jdk -q
```

Comment:
This repo has _a lot_ of commits. Do we want any shallowness?

Reply:
Shallowness in `jdk` is less likely to cause problems, so maybe it's worth keeping shallow.

Reply:
What about single-branch?